.travis.yml

```diff
@@ -14,9 +14,12 @@ notifications:
         on_failure: always
 
 install:
-    - "pip install -r requirements/base.txt --use-mirrors"
-    - "pip install -r requirements/tests.txt --use-mirrors"
-    - pip install coveralls
+    - "pip install pip --upgrade"
+    - "pip install -r requirements/base.txt"
+    - "pip install -r requirements/2nd-pass.txt"
+    - "pip install -r requirements/tests.txt"
+    - "yes | pip uninstall South || true"
```

Comment:
This seems like this should not be needed. The env should always be recreated when started starts a new build, if for some reason we want to remove a dependency, I guess we can simply purge the cache.